tests/test_setup_door.py

```
@@ -465,3 +628,329 @@ def test_new_planning_scaffolds_and_records(self, tmp_path):
+        dotenv = target / ".env"
+        assert dotenv.is_file()
+        assert (dotenv.stat().st_mode & 0o777) == 0o600
+        assert SECRET in dotenv.read_text(encoding="utf-8")
+        assert SECRET not in result.stdout + result.stderr
+        check_ignore = subprocess.run(
+            ["git", "-C", str(target), "check-ignore", "-q", ".env"],
+            env=env,
+            timeout=30,
+        )
+        assert check_ignore.returncode == 0, ".env must be gitignored"
+        status = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)